packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/emitting/NotificationsEmitter.kt

```diff
@@ -46,6 +46,15 @@ open class NotificationsEmitter : Module(), NotificationListener {
       lastNotificationResponseBundle = null
       null
     }
+
+    Function<Bundle?>("getLastNotificationResponse") {
+      lastNotificationResponseBundle
```

Comment:
let's take it a step further here: let's remove the async functions in the native modules, and have the async functions on the JS side call the sync functions.

Reply:
@douglowder the async functions are still there

Reply:
Thanks @vonovak -- I do see that now. I'll back out my suggestions. I do see that the availability check was removed for the async functions, and that needs to be added back. I'll add some suggestions there.